content/metadata.yaml

```
@@ -80,6 +80,20 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2020-03-22
+  -   
+    github: RLordan
+    name: Ronan Lordan
+    initials: RL
+    orcid: 0000-0001-9668-3368
+    twitter: @el_ronan
```

```suggestion
    twitter: el_ronan
```

Hopefully this will fix the build error